src/graph/interface/logical_tensor.cpp

```diff
@@ -177,6 +177,9 @@ size_t logical_tensor_wrapper_t::hash() const noexcept {
             break;
         default: assertm(false, "unknown layout_type");
     }
+    // property type
+    if (this->property_type() != property_type::undef)
+        seed = hash_combine(seed, static_cast<size_t>(this->property_type()));
```

Comment:
I think we don't need the check and always hash the property type?

Reply:
I was under the impression that this undef property will be changed later, so I didn't hash it. But after double check, there's no change for this property, so yes, I agree we don't need the check.